frameworks/elastic/universe/config.json

```diff
@@ -1060,6 +1060,26 @@
           "description": "[DEPRECATED] Starting from Elasticseach 6.0, this setting is no longer configurable and defaults to true. Changing this setting has no effect.",
           "type": "boolean",
           "default": true
+        },
+        "xpack_monitoring_enabled": {
+          "description": "X-Pack monitoring is enabled by default on Elastic 6.3+. Changing this setting has no effect.",
+          "type": "boolean",
+          "default": true
+        },
+        "xpack_ml_enabled": {
+          "description": "X-Pack ml is enabled by default on Elastic 6.3+. Changing this setting has no effect.",
+          "type": "boolean",
+          "default": true
+        },
+        "xpack_watcher_enabled": {
+          "description": "X-Pack ml is enabled by default on Elastic 6.3+. Changing this setting has no effect.",
+          "type": "boolean",
+          "default": true
+        },
+        "xpack_reporting_enabled": {
+          "description": "X-Pack Monitoring us enabled by default on Elastic 6.3+. Changing this setting has no effect.",
+          "type": "boolean",
+          "default": true
```

Comment:
The descriptions can be improved. Also, make sure that the default values for our settings mirror those of Elasticsearch. For example, it looks like X-Pack monitoring is disabled by default on Elasticsearch.

Reply:
Yes, AFAIK the settings that I have included are enabled by default.
Please have a look here: https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html

Reply:
It looks like the default values for all but `xpack.reporting.enabled` is `true`:
```
$ dcos task exec elastic__master-0-node__e2d370c5-d916-48c1-8630-7e4f8fbfe3f5 /opt/mesosphere/bin/curl -ki -H 'Content-type: application/json' 'http://coordinator.elastic.l4lb.thisdcos.directory:9200/_cluster/settings?include_defaults=true&format=yaml&flat_settings=true' | grep 'xpack.monitoring.enabled\|xpack.ml.enabled\|xpack.watcher.enabled\|xpack.monitoring.enabled'
  xpack.ml.enabled: "true"
  xpack.monitoring.enabled: "true"
  xpack.watcher.enabled: "true"
```

`xpack.reporting.enabled` seems to be a Kibana-only setting, so we can remove it from here.

Reply:
Okay, let's remove it. Thanks :)

Reply:
Hey @mpereira !Can you have a look at it?